index.js

```diff
@@ -4,13 +4,42 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "red"
 
 /***** Deliverable 3 *****/
-// console.log("the player object looks like this:", player)
+ console.log("the player object looks like this:", player)
+const playerContainer = document.querySelector(".player")
+const playerImg = document.querySelector("img")
+const playerName = document.querySelector("h2")
+const playerNickname = document.querySelector("em")
+playerImg.src = player.photo
+playerImg.alt = player.name
+playerName.textContent = player.name
+playerNickname.textContent = player.nickname
```

Comment:
It might be helpful to wrap this up into a function to clean up your code, and make it a bit more reusable - but overall this looks good!